parallel_algorithms/P3179/P3179.md

```diff
@@ -1287,10 +1306,10 @@ via an uncaught exception, the behavior is determined by the [`ExecutionPolicy`]
 
 ## Modify [algorithms.parallel.overloads]{- .sref} ## {#modify_alg_parallel_overloads}
 
-[1]{.pnum} Parallel algorithms are algorithm overloads. Each parallel algorithm overload has an additional template
-type parameter named `ExecutionPolicy` [or `@_execution-policy_@ Ep`]{.add}, which is the first template parameter.
-Additionally, each parallel algorithm overload has an additional function parameter of type `ExecutionPolicy&&`
-[or `Ep&&`]{.add}, which is the first function parameter.
+[1]{.pnum} Parallel algorithms are algorithm overloads. [Each parallel algorithm overload has an additional template
+type parameter named `ExecutionPolicy`, which is the first template parameter. Additionally, e]{.rm}[E]{.add}ach parallel
```

Comment:
```suggestion
[1]{.pnum} Parallel algorithms are algorithm overloads. Each [parallel algorithm overload has an additional template
type parameter named `ExecutionPolicy`, which is the first template parameter. Additionally, each]{.rm} parallel
```
Same new wording, just reuses the already capitalized "Each" from the previous sentence instead of changing the case.

Reply:
Rewritten a little bit differently. Let me know if you still don't like it